src/devices/Mcp3xxx/Mcp3304.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Device.Spi;
+
+namespace Iot.Device.Adc
+{
+    /// <summary>
+    /// MCP3304 Analog to Digital Converter (ADC)
+    /// </summary>
+    public class Mcp3304 : Mcp33xx
+    {
+        /// <summary>
+        /// Constructs Mcp3302 instance
+        /// </summary>
+        /// <param name="spiDevice">Device used for SPI communication</param>
+        public Mcp3304(SpiDevice spiDevice) : base(spiDevice, channelCount: 8, adcResolutionBits: 13) { }
```

Comment:
this one is unsigned I presume (curious because previous 13-bit was signed)

Reply:
With the 33 series if you are reading single ended it’s really only 12 bit positive whic I suppose is the same as 13 bit signed with the sign bit as 0. Is that what you were asking

Reply:
@Frankenslag not sure - I know Mcp3008 starts sending reading bits backward

Reply:
Note... for the 3301 I have changed the name of the function used to read the value to ReadDifferential (13 bit signed) as the 3301 does not have the ability to read single ended.